frappe_graphql/frappe_graphql/mutations/add_translation.py

```diff
@@ -0,0 +1,37 @@
+from graphql import GraphQLSchema, GraphQLResolveInfo
+
+import frappe
+
+
+def bind(schema: GraphQLSchema):
+    schema.mutation_type.fields["addTranslation"].resolve = add_translation_resolver
+
+
+def add_translation_resolver(obj, info: GraphQLResolveInfo, **kwargs):
+    language = kwargs.get("language")
+    source_text = kwargs.get("source_text")
+    translated_text = kwargs.get("translated_text")
+    context = kwargs.get("context")
+    doctype = kwargs.get("doctype")
+    docname = kwargs.get("docname")
+    docfield = kwargs.get("docfield")
+
+    context = None
```

Comment:
@faztp12 Since this is set to None, let's remove context as a param?

Reply:
@assemmarwan um, it could be helpful for translations outside doctypes/documents right ? i pushed another commit fixing this